### ISSUE TITLE
Update filter expression page to include filtering by array

### ISSUE THREAD

**Issue by bendangelo** [meilisearch/documentation#3084]
I didn't know you could index an array and then filter by multiple tags like so:

```
currencyTags = EUR AND countryTags = BANGLADESH
```
I think this should be added to this page: https://www.meilisearch.com/docs/learn/filtering_and_sorting/filter_expression_reference

**Comment by guimachiavelli** [comment]
Hello and thanks for opening an issue in this repo, @bendangelo, it's much appreciated.

I'm not sure I completely understand what you mean. Were you surprised by the fact the `=` operator matches values inside an array, such as e.g. `currencyTags = EUR` returning a document with `currencyTags = [ "EUR"]`?

**Comment by bendangelo** [comment]
Yes exactly. I didn't know it was possible to use on array values.
